cmd/meta/main.go

```diff
@@ -188,16 +195,23 @@ func (m *Main) RunDump(args Args) error {
 }
 
 func (m *Main) RunServer(args Args) error {
-	var musicbrainzDB *sql.DB
+	var DB *sql.DB
 	if uri := args.String("--musicbrainz-index"); uri != "" {
 		db, err := sql.Open("sqlite3", uri)
 		if err != nil {
 			return err
 		}
 		defer db.Close()
-		musicbrainzDB = db
+		DB = db
+	} else if uri := args.String("--cwr-index"); uri != "" {
```

Comment:
We should support starting the server with both MusicBrainz and CWR indexes.

Reply:
@lmars 
Are you referring to supporting  a command like : 
meta server --musicbrainz-index artist.db --cwr-index registeredwork.db